src/test/java/io/github/jamsesso/jsonlogic/InExpressionTests.java

```diff
@@ -40,7 +41,7 @@ public void testArrayNotIn() throws JsonLogicException {
 
   @Test
   public void testInVariableInt() throws JsonLogicException {
-    Map data = Collections.singletonMap("list", Arrays.asList(1, 2, 3));
+    Map data = Collections.singletonMap("list", Arrays.asList(BigDecimal.ONE, new BigDecimal("2"), new BigDecimal("3")));
```

Comment:
Lets keep the existing tests and add new tests with BigDecimal separately

Reply:
Okay